include/glow/Importer/ModelLoader.h

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef GLOW_IMPORTER_MODELLOADER_H
```

Comment:
nit: #pragma once is shorter and avoid potential issues with misspelled string in def/ifndef.

Reply:
I think for historic reasons. @nadavrot any insights into this?

Reply:
`#pragma once` is non-standard (http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf8-use-include-guards-for-all-h-files), so many projects use include guards for that reason.

Reply:
Personally, I like the use of ifdefs, mainly because LLVM uses it. I am not so worried about typos because clang-tidy has a checker that ensures that the file path is used as header guard.

Reply:
leaving ifdefs as is then